transformer_engine/pytorch/fp8.py

```diff
@@ -536,7 +536,7 @@ def global_amax_reduction(
     wait_handle = reduce_tensor_across_group_op_max(
         contiguous_amax,
         fp8_meta["fp8_group"],
-        fp8_meta["async_amax_reduction"],
+        fp8_meta["async_op"],
```

Comment:
I'm nitpicking, but this variable was better off before, i.e. "async_amax_reduction". My name change suggestion was specifically for the `reduce_tensor_across_group_op_max` function because it that function not specifically related to amax reduction.